### DISCUSSION TITLE
Upgrading Github Enterprise 3.8.3 to 3.14

### DISCUSSION THREAD

**Discussion by Radiant1Can** [community/community#144450] [Enterprise]
### Select Topic Area

Question

### Body

Hi. I inherited a Github Enterprise server v3.8.3 and need to upgrade. It says I have to upgrade to 3.10, then 3.12 then 3.14, which is fine, but when I try to upgrade from 3.8.3 to 3.10.17 it gives the error:  You are upgrading to 3.9 or later which requires version >= v3.7.14 or >= v3.8.7.

I cannot upgrade to 3.8.7 or higher, except for 3.10.17 as anything else is no longer available. Does anyone have any suggestions?

**Comment by Radiant1Can** [comment]
I was able to get an upgrade package that works.

**Comment by djdefi** [comment]
Glad to hear you were able to find the right package!

The [GitHub Enterprise Server upgrade assistant](https://support.github.com/enterprise/server-upgrade) may be useful for future multiple version upgrades and obtaining upgrade packages. Additionally, GitHub Enterprise support can help provide downloads for retired releases if needed.
